dot-net/Centroid/Config.cs

```diff
@@ -78,7 +78,23 @@ public override bool TryConvert(ConvertBinder binder, out object result)
 
             try
             {
-                result = Convert.ChangeType(RawConfig, binder.Type);
+                JToken rawConfig = RawConfig;
+                result = rawConfig.ToObject(binder.Type);
+                //ensure at least one property in the config bound to the type
```

Comment:
If we're just letting it do its thing, do we need any more than the already included 2 tests?